testing/test.cpp.ib

```diff
@@ -2,20 +2,19 @@
 
 class test
 	public:
-		test(int hidden):_hidden(hidden)
-			
-		
+		test(int hidden):_hidden(hidden) {}
+
```

Comment:
We should not rely on two empty lines after the constructor, because IIRC `test();` might be valid. C++ is weird.